tests/aiger/and_bad.aag

```diff
@@ -0,0 +1,8 @@
+aag 3 2 0 0 1 1 0 0 0
```

Comment:
Oh yeah in hindsight i did not give that file a very good name. its just an aiger file that has a bad state node instead of an output node (so would have caused the issue that this pr fixes). it is a valid aiger file though. will rename

Reply:
Yeah please do other than that the PR looks good